src/apps/frontend/app.component.tsx

```diff
@@ -19,10 +19,10 @@ export default function App(): React.ReactElement {
   return (
     <AuthProvider>
       <AccountProvider>
-        <Toaster />
-        <Router>
-          <AppRoutes />
-        </Router>
+          <Toaster />
```

Comment:
Kindly undo unrelated formatting changes here and other places

Reply:
sure